core/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -943,7 +952,7 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeOptionalString(routing);
             out.writeOptionalString(parent);
             out.writeLong(version);
-            
+
             out.writeByte(versionType.getValue());
             out.writeLong(autoGeneratedIdTimestamp);
             out.writeVLong(seqNo);
```

Comment:
I discussed this with @bleskes via another channel. Our current theory is that assertions are not enabled for standalone nodes running in our integration tests. I will investigate and address accordingly.